packages/fakeit-file-loader/package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I don't understand the difference between `@fakeit/file-loader` and `@fakeit/input` and why there needs to be both because it seems like they do the same thing.